.mergify.yml

```diff
@@ -13,3 +13,17 @@ pull_request_rules:
     actions:
       queue:
         name: default
+        
+  - name: Automatic merge on approval (Picasso Runtime)
+    conditions:
+      - or:
+        - "#approved-reviews-by>=3"     
```

Comment:
I would remove this as without a review from core or technical council, Picasso runtime must not be updated

Reply:
But we can include a "or" condition to require review from technical council members

Reply:
You may update it in next PR. I did one step stricter. I do not do it as strict as possible. That is out of scope. So I do one step to improve. 

I am sure with my PR it is better then without. So it is not ideal. But still better. 

Settling ` technical council members` in git is out of scope of this PR and afaik not planned. So after it is done - feel free to change the rule.

Reply:
Suggesting using `core` only is good, but risky too.

Issue and required task with tech council members and core team mentioned in slack and meeting several times by me. So I would not do it.

Reply:
>Suggesting using core only is good, but risky too.

The risk is worth taking for now IMO, as it has 3 of the members of the technical council now. I can add a condition later to require technical council review later.

Reply:
Imho I do not want to be person indulging work stoppage on people works by thinking that 3 tech member are really review this stuff (2 are likely not) - so only one and he is busy.   So it is risky. Also core has non overlapping meaning with tech collective. So it is not 1 = 1.

Reply:
Neither want to jump ahead and even badly assembled tech oracle. Final design is

1. each tech collective member must have identity (it can be nick name) and each such must have gpg key
2. any change to runtime triggers 
2.1. subkey dumps gpg keys form identity pallet
2.2. git checks gpg signatures on PR/branch/tag
2.3 if these do not apply - it fails

So whatever we can do to mergify will not replicate what is final achievable solution.

So current PR does improve things, but not blocks work. 

@vimukthi-git do you recall that we must gpg sign release? So actually we can avoid that action or reduce it security and embeeed into release pipeline same way we do with runtime changes.

But I do not want to spent time for issues. I just wanted to  do some more realistic rules and increase security in 30 minutes of work :) For any other improvements (with any changes risking blocking work done) it should go via CU.

Reply:
cc @andyjsbell just FYI . above some design for continuous improvement of our release and PR process (with automation). I did not digged GitLab runners of Parity, but at least per their release docs and tooling they are starting(ed) beginning of like this.

Reply:
I think this is good @vimukthi-git, if you are happy then let's merge this one

Reply:
@dzmitry-lahoda validation on the release by and signing it is not the time to find out that things are horribly broken. PR should allow catching any problems with the specific change as much as possible. I'm ok with the current change though.